packages/expo-dev-launcher/android/src/debug/java/expo/modules/devlauncher/modules/DevLauncherDevMenuExtensions.kt

```diff
@@ -10,24 +10,28 @@ import expo.interfaces.devmenu.items.DevMenuItemsContainer
 import expo.interfaces.devmenu.items.DevMenuItemsContainerInterface
 import expo.interfaces.devmenu.items.KeyCommand
 import expo.modules.devlauncher.DevLauncherController
-import expo.modules.devlauncher.DevLauncherController.Companion.instance
+import expo.modules.devlauncher.koin.devLauncherKoin
+import expo.modules.devlauncher.launcher.DevLauncherControllerInterface
 
 class DevLauncherDevMenuExtensions(
   reactContext: ReactApplicationContext?
 ) : ReactContextBaseJavaModule(reactContext),
   DevMenuExtensionInterface {
+
   override fun getName(): String {
     return "ExpoDevLauncherDevMenuExtensions"
   }
 
   override fun devMenuItems(settings: DevMenuExtensionSettingsInterface): DevMenuItemsContainerInterface =
     DevMenuItemsContainer.export {
-      if (!DevLauncherController.wasInitialized() || instance.mode == DevLauncherController.Mode.LAUNCHER) {
+      val controller = devLauncherKoin().getOrNull<DevLauncherControllerInterface>()
+
+      if (controller?.mode == DevLauncherController.Mode.LAUNCHER) {
```

Comment:
```suggestion
      if (controller == null || controller?.mode == DevLauncherController.Mode.LAUNCHER) {
```
to match the existing behavior? (though it seems like this case shouldn't ever happen anyway)

Reply:
It matches the existing behavior. If the controller is null, controller?.mode will also be null. So it won't be equal to `DevLauncherController.Mode.LAUNCHER`